activities/views/follows.py

```diff
@@ -15,19 +17,25 @@ class FollowsPage(TemplateView):
 
     def get_context_data(self):
         # Gather all identities with a following relationship with us
-        identities = {}
-        for outbound_follow in self.request.identity.outbound_follows.filter(
-            state__in=FollowStates.group_active()
-        ):
-            identities.setdefault(outbound_follow.target, {})[
-                "outbound"
-            ] = outbound_follow
-        for inbound_follow in self.request.identity.inbound_follows.filter(
-            state__in=FollowStates.group_active()
-        ):
-            identities.setdefault(inbound_follow.source, {})["inbound"] = inbound_follow
+        follows = Follow.objects.filter(
+            Q(source=self.request.identity) | Q(target=self.request.identity),
+            state__in=FollowStates.group_active(),
+        ).order_by("-id")
```

Comment:
There's a "created" on Follow too, let's order by that instead.

Reply:
Done at https://github.com/jointakahe/takahe/pull/170/commits/265ff7c50f3c261ef858fa61c46a0466dea5d49c